docs/source/tools/ledger-api-test-tool/index.rst

```diff
@@ -0,0 +1,127 @@
+at a port :code:`<port>`:
+
+    ``ledger-api-test-tool -h <host> -p <port>``
+
+For example
+
+    ``ledger-api-test-tool -h localhost -p 6865``
+
+Exploring options the tool provides
+===================================
+
+Run the tool with :code:`--help` flag to obtain the list of options the tool provides:
+
+   .. code-block:: console
+
+     $ ledger-api-test-tool --help
+
+     The Ledger API Test Tool is a command line tool for testing the correctness of
+     ledger implementations based on DAML and Ledger API.
+     Usage: ledger-api-test-tool [options]
+
+       --help                   prints this usage text
+       -p, --target-port <value>
+                                Ledger API server port. Defaults to 6865.
+       -h, --host <value>       Ledger API server host. Defaults to localhost.
+       --must-fail              One or more of the scenario tests must fail. Defaults to false.
+       -r, --reset              Perform a ledger reset before running the tests. Defaults to false.
+       -x, --extract            Extract the testing archive files and exit.
+
+Try out the Ledger API Test Tool against DAML Sandbox
+=====================================================
+
+To run the tool against [DAML
+Sandbox](https://docs.daml.com/tools/sandbox.html), run:
+
+   .. code-block:: console
+
+     ledger-api-test-tool -x
+     da sandbox -- SemanticTests.dar
+     ledger-api-test-tool
+
+This should always succeed! This is useful if you do not have yet a custom
+Ledger API endpoint.
+
+The DAML Sandbox starts at by default at :code:`localhost:6865`
+and the Ledger API Test Tool uses this as the default endpoint to test, hence
+hosts and ports command line arguments can be omitted.
+
+Testing your tool from continuous integration pipelines
+=======================================================
+
+To test your ledger in your CI pipelines, execute it as part of your pipeline:
+
+
+   .. code-block:: console
+
+     $ ledger-api-test-tool
+     Running 10 scenarios against localhost:6865...
+     Testing scenario: Test:timeTravel
+     Testing scenario: Test:authorization_rule1_failure
+     Testing scenario: Test:authorization_success2_delegation
+     Testing scenario: Test:authorization_rule2_failure
+     Testing scenario: Test:startsAtEpoch
+     Testing scenario: Test:authorization_success1
+     Testing scenario: Test:privacy_projections1
+     Testing scenario: Test:consistency_doublespend1
+     Testing scenario: Test:consistency_doublespend3
+     Testing scenario: Test:consistency_doublespend2
+     All scenarios completed.
+     $ echo $?
+     0
+
+The tool is tailored to be used in CI pipelines: as customary, when the tests
+succeed, it will produce minimal output and return the success exit code.
+
+Using the tool with a known-to-be-faulty Ledger API implementation
+==================================================================
+
+To force the tool to always return success exit code, use :code:`--must-fail` flag:
```

Comment:
The `must-fail` option was introduced for a different purpose: it should allow checking that the tool actually does successfully detect a failure.

A name that better captures this behaviour is `internal-expect-at-least-one-failure`. This option is not meant to be used by consumers.

Suggestion:
- explain that with the `must-fail` option the tool returns success on at least one failure, and fails if there no failure
- add explanation that we use this internally to test that the tool does not always return true
- add link to an issue for specifying the names (or patterns of names) of the tests that are expected to fail, which is the generalization that is actually useful for DAML ledger developers

Reply:
Simon's explanation is very helpful here - it really helps to capture the purpose. 

That explains why we need it internally. Before you've implemented the feature to specify the tests that will fail, what's the value of this for external users?

Reply:
I think `Using the tool with a known-to-be-faulty Ledger API implementation` captures the value of this option for the external users.

@meiersi-da, I do not understand what you mean in 
```
add link to an issue for specifying the names (or patterns of names) of the tests that are expected to fail, which is the generalization that is actually useful for DAML ledger developers
```

I've tweaked the section.